docs/components/webapp.rst

```diff
@@ -54,12 +54,9 @@ superuser.
 
 Run this::
 
-  $ docker-compose run webapp python manage.py createsuperuser
+  $ docker-compose run webapp python manage.py makesuperuser email@example.com
```

Comment:
This is good, but there's a bug in that when you run the webapp container, the cwd is `/app/` and it needs to be `/app/webapp-django/`. So this line should be:

```
$ docker-compose run webapp python webapp-django/manage.py makesuperuser willkg@mozilla.com
```

Can you fix that in this PR? If you don't want to, that's totally fine--I'll do a follow-up PR.